training.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: training
+title: Training Materials
+---
+
+# Foreman training materials
+
+This page lists the materials available for those wishing to study/practice working
+with Foreman, as an alternative style to reading the manual. 
+
+## Netways Foreman Training Course
+        
+This course was developed by Netways and made available to the Foreman
+community for use in developing training courses of their own, or for community
+members to use for self-study. It is designed as a two-day hands-on training
+course, and as well as the presentation below, it comes with
+[handouts][handouts], [exercises][exercises], and [solutions][solutions].
+
+[handouts]: https://github.com/NETWAYS/foreman-training/releases/download/v1.0/foreman-training-handouts.pdf
+[exercises]: https://github.com/NETWAYS/foreman-training/releases/download/v1.0/foreman-training-exercises.pdf
+[solutions]: https://github.com/NETWAYS/foreman-training/releases/download/v1.0/foreman-training-solutions.pdf
+
+A full-page version of the presentation can be found at
+[netways.github.io/foreman-training/static][rendered]
+
+[rendered]: //netways.github.io/foreman-training/static
```

Comment:
This renders tiny on a FullHD resolution, can you make it bigger? 
![screenshot from 2016-05-04 16-25-48](https://cloud.githubusercontent.com/assets/598891/15017349/87da0de0-1215-11e6-8956-c852a79bcdb5.png)